packages/ses/test/error/tame-console-unsafeError.test.js

```diff
@@ -42,7 +42,7 @@ test('assert - safe', t => {
   } catch (barErr) {
     console.error('bar happens', barErr);
   }
-  t.end();
+  t.pass();
```

Comment:
Generally you've simply been omitting `t.end()`. Why do you need to replace it with a `t.pass()` here (and a few other places?)

Reply:
With Ava, every test must have at least one assertion, or call pass at the end. Tests that might not call an assertion method in the success path need to call this expressly.